dashboard/test/ui/features/star_labs/ailab.feature

```diff
@@ -0,0 +1,56 @@
+Feature: AI Lab
+
+  @as_student
+  Scenario: Training a model with an in-house dataset
+    Given I am on "http://studio.code.org/s/allthethings/lessons/43/levels/1"
+    And I wait to see the dynamic instructions contain "selectDataset" key
+    And I close the instructions overlay if it exists
+    And I click selector ".ailab-image-hover"
```

Comment:
I think we should be more explicit about what we are clicking here.

Reply:
Also is this test at risk if the datasets or their order changes?

Reply:
I added an `ailab-dataset-image` class for the dataset images. Is that what you had in mind?

I don't believe the test would be at risk if the datasets or order changes. The test doesn't specify anything about a particular dataset and it looks like it chooses the first one on the page.